### DISCUSSION TITLE
CNN vs. Vision Transformer: A Practitioner’s Guide to Selecting the Right Model

### DISCUSSION THREAD

**Discussion by giscus** [tobiasvanderwerff/tobiasvanderwerff.github.io#1] [Announcements]
# CNN vs. Vision Transformer: A Practitioner’s Guide to Selecting the Right Model

Vision Transformers (ViTs) have become a popular model architecture in computer vision research, excelling in a variety of tasks and surpassing Convolutional Neural Networks (CNNs) in most benchmarks. As practitioners, we often face the dilemma of choosing the right architecture for our projects. This blog post aims to provide guidelines for making an informed decision on when to use CNNs versus ViTs, backed by empirical evidence and practical considerations.

https://tobiasvanderwerff.github.io/2024/05/15/cnn-vs-vit.html

**Comment by lricardo09** [comment]
Very helpful! Thank you for taking your time to write this! By the way, is part 2 coming out?

**Reply by tobiasvanderwerff** [reply]
Glad to hear it was helpful! Regarding a potential part 2: I'm currently working on other topics, so it's not something I'm currently planning to do. I might consider it if I find myself diving into this material again in the future.
